src/pages/signup/style.ts

```
@@ -0,0 +1,93 @@
+  width: 18px;
+  height: 18px;
+  accent-color: ${theme.color.main[500]};
+  cursor: pointer;
+`;
+
+export const DropdownRow = styled.div`
+  display: flex;
+  gap: 12px;
+`;
+
+export const Error = styled.div`
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)